### PR TITLE
Add ability to select QSV device in the UI

### PR DESCRIPTION
**Pull request by evanjarrett**
**Changes**

This provides an input field to specify the device used for QSV rendering.

**Issues**
This depends on the backend PR here: https://github.com/jellyfin/jellyfin/pull/12384

### CODE REVIEW

File: src/strings/en-us.json [hunk 1]
```diff
@@ -789,6 +789,8 @@
     "LabelPublicHttpsPortHelp": "The public port number that should be mapped to the local HTTPS port.",
     "LabelPublishedServerUri": "Published Server URIs",
     "LabelPublishedServerUriHelp": "Override the URI used by Jellyfin, based on the interface, or client IP address.",
+    "LabelQsvDevice": "QSV Device",
+    "LabelQsvDeviceHelp": "This is the render node that is used for hardware acceleration.",
```
Comment (nyanmisaka) [inline src/strings/en-us.json:793]: Please add a slightly more detailed description of what the user should fill in. And emphasize that this should be left blank by default.

Comment (evanjarrett) [reply src/strings/en-us.json:793]: Hows something like this?

"LabelQsvDeviceHelp": "Specify the device for Intel QSV. On Linux, this is the render node, e.g., /dev/dri/renderD128. On Windows, this is the device index starting from 0. Leave blank to use the default device."

Comment (nyanmisaka) [reply src/strings/en-us.json:793]: `"LabelQsvDeviceHelp": "Specify the device for Intel QSV on a multi-GPU system. On Linux, this is the render node, e.g., /dev/dri/renderD128. On Windows, this is the device index starting from 0. Leave blank unless you know what you are doing."`

Made some changes to keep away newbie users from touching this.

**Comment by Bond-009** [comment]
Server PR is now merged

**Comment by nyanmisaka** [comment]
@thornbill I think this is no longer blocked by the server and can be merged to 10.10.

**Review by thornbill — APPROVED** [review]

**Pull request merged**